README.md

```diff
@@ -59,12 +59,16 @@ You can then use the following additional functions on Dataset/DataFrame
   Solve temporal overlaps by a prioritizing Records according to rnkExpressions and extend the temporal range of each key to be defined over the whole timeline. The resulting DataFrame has an additional column `_defined` which is false for extended ranges.
   - aggExpressions: Aggregates to be calculated on overlapping records (e.g. count)
   - rnkFilter: Flag if overlapping records should be tagged or filtered (default=filtered=true)
-- `temporalCombine( keys:Seq[String] = Seq(), ignoreColNames:Seq[String] = Seq() )(implicit ss:SparkSession, hc:TemporalQueryConfig)`
+- `temporalCombine( keys:Seq[String] = Seq(), ignoreColNames:Seq[String] = Seq() )`
   Combines successive records if there are no changes on the non-technical attributes.
   **The parameter `keys` is superfluous. Please refrain from using it!**
   It is still present in order not to break the API.
   - ignoreColName: A list of columns to be ignored in change detection
-- `temporalUnifyRanges( keys:Seq[String] )(implicit ss:SparkSession, hc:TemporalQueryConfig)`
+- `temporalUnifyRanges( keys:Seq[String] )`
   Unify temporal ranges in group of records defined by 'keys' (needed for temporal aggregations).
 - `temporalExtendRange( keys:Seq[String], extendMin:Boolean=true, extendMax:Boolean=true )`
   Extend temporal range to min/maxDate according to TemporalQueryConfig
+- `temporalContinuous2discrete`
+  sets the discreteness of the time scale to milliseconds
```

Comment:
I think the text of the two functions should be switched, isn't it?

Reply:
Corrected: https://github.com/zzeekk/spark-temporalquery/pull/14/commits/236fe3c226e673edc7c4c55915127f7477e2170e